app/assets/javascripts/api_stub.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Created by kaceyryan on 5/2/15.
+ */
+
+var request = require('request');
+
+class API {
+  //get images in JSON object for your location
+  getImages(callback) {
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(function (position) {
+        var query = "/images?lat=".concat(position.coords.latitude, "&long=", position.coords.longitude);
+        //var query = "http://localhost:3000/images?lat=34.0722&long=118.4441";
+        request(query, function (error, response, body) {
+          if (!error && response.statusCode == 200) {
+            callback(JSON.parse(body));
+          }
+        });
+      }, this.showError);
+    } else {
+      //Handle error
+      //x.innerHTML = "Geolocation is not supported by this browser.";
+    }
+  }
+
+  //post an image for your provided location
+  //Does this need a callback?
+  postImage(image, callback){
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(function (position) {
+        var query = "/images?lat=".concat(position.coords.latitude, "&long=", position.coords.longitude);
```

Comment:
`request` makes constructing query strings really easy so that we don't have to reinvent the wheel. We can do:

```js
request.get(
  {
    uri: '/images',
    qs: {
      lat: position.coords.latitude,
      long: position.coords.longitude,
    }
  },
  function() {}
);
```